shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/util/SQLUtil.java

```diff
@@ -95,6 +95,9 @@ public static String getExpressionWithoutOutsideParentheses(final String value)
     
     private static int getParenthesesOffset(final String value) {
         int result = 0;
+        if(Strings.isNullOrEmpty(value)){
+            return result;
+        }
```

Comment:
A unit test is expected to add in `org.apache.shardingsphere.sql.parser.sql.util.SQLUtilTest`.